src/shared/ui/paragraph/paragraph.scss

```diff
@@ -1,12 +1,12 @@
 .paragraph {
   @include media-laptop {
-    font-size: 16px;
-    line-height: 24px;
+    font-size: 14px;
+    line-height: 18px;
   }
 
-  font-size: 18px;
+  font-size: 14px;
   font-weight: $font-weight-regular;
-  line-height: 28px;
+  line-height: 18px;
```

Comment:
Why did you change font size to `14px`? I don't see any requirment for that in the task